File: C++/اعداد فیثاغورثی.cpp
#include <iostream>
using namespace std;
int main()
{
    int a, b, c, temp;
    cin >> a;
    cin >> b;
    cin >> c;
    if (a < b)
    {
        temp = b;
        b = a;
        a = temp;
    }
    if (a < c)
    {
        temp = c;
        c = a;
        a = temp;
    }
    if (c * c + b * b == a * a)
        cout << "YES";
    else
        cout << "NO";
}

File: Python/اعداد فیثاغورثی.py
a = int(input())
b = int(input())
c = int(input())
if a < b:
    a, b = b, a
if a < c:
    a, c = c, a
if c ** 2 + b ** 2 == a ** 2:
    print("YES")
else:
    print("NO")